CMakeLists.txt

```diff
@@ -60,15 +60,17 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 
 # Use CMAKE_<LANG>_COMPILER_LAUNCHER if available.
-find_program(SCCACHE_EXECUTABLE sccache)
-mark_as_advanced(SCCACHE_EXECUTABLE)
-if(SCCACHE_EXECUTABLE)
-  foreach(LANG CXX C)
-    if(NOT DEFINED CMAKE_${LANG}_COMPILER_LAUNCHER AND NOT CMAKE_${LANG}_COMPILER MATCHES ".*/s?ccache$")
-      message(STATUS "Enabling sccache for ${LANG}")
-      set(CMAKE_${LANG}_COMPILER_LAUNCHER ${SCCACHE_EXECUTABLE} CACHE STRING "")
-    endif()
-  endforeach()
+if(NOT MSVC)
```

Comment:
question, looks like if it is MSVC, we are not going to configure sccache? Or is this configured somewhere else?

Reply:
From the build log, I think it is configed in build env for PyTorch. Gloo will leverage sccache-cl.exe to compile, just as other submodules in PyTorch.

Reply:
Looks like all that is needed here is to add another  `CMAKE_${LANG}_COMPILER` pattern for Windows, wouldn't it?

Reply:
Yes, you are right. For windows it is a sccache-cl.exe. Resolved it.